test/e2e/servers/ory/oathkeeper.spec.ts

```diff
@@ -72,6 +73,14 @@ describe("Oathkeeper graphql endpoints", () => {
 })
 
 describe("idempotencyMiddleware", () => {
+  gql`
```

Comment:
don't understand why this is coming in

Reply:
It was deleted from galoy-pay and so there was no definition left for the generated.ts file created from `make codegen` so I needed to add it back here since it's still used here